browser/ui/brave_pages.cc

```diff
@@ -29,4 +32,24 @@ void ShowBraveSync(Browser* browser) {
       GetSingletonTabNavigateParams(browser, GURL(kBraveUISyncURL)));
 }
 
+void ShowClearRewardsDataDialog(Browser* browser) {
```

Comment:
why are there 3 methods here? `ShowClearRewardsDataDialog`, `ShowSettingsSubPage` and `ShowSettingsSubPageInTabbedBrowser`
I don't see anyplace else that calls ShowSettingsSubPage or ShowSettingsSubPageInTabbedBrowser

Reply:
actually what calls `ShowClearRewardsDataDialog`?

Reply:
These are leftovers. Removed